src/pytest_benchmark/stats.py

```diff
@@ -35,38 +41,101 @@ def as_dict(self):
     def update(self, duration):
         self.data.append(duration)
 
-    @cached_property
+    @property
     def sorted_data(self):
         return sorted(self.data)
 
-    @cached_property
-    def total(self):
+    @property
+    def total(self) -> float or int:
+        """ Return the total time of round / iterations."""
+        cache_value = self.cache.get('total')
+
+        if cache_value is not None:
+            return cache_value
+
         return sum(self.data)
```

Comment:
Shouldn't we store this data in the cache?

Reply:
Yes, I didn't see that this variable was not affected by my modifications. (I did CTRL + F)